railib/rest.py

```diff
@@ -119,13 +121,52 @@ def _print_request(req: Request, level=0):
                 print(json.dumps(json.loads(req.data.decode("utf8")), indent=2))
 
 
+def _cache_file() -> str:
+    return path.join(path.expanduser('~'), '.rai', 'tokens.json')
+
+
+# Read oauth cache
+def _read_cache() -> dict:
+    try:
+        with open(_cache_file(), 'r') as cache:
+            return json.loads(cache.read())
+    except Exception:
+        return None
```

Comment:
I see we swallow these exceptions in these cache functions (the go sdk does sth similar as well). Is that on purpose?

Reply:
as far as I do understand, if the cache is not accessible for some reason we should not block the user and instead we request a new token